lab/SoundTest/SoundTest.js

```diff
@@ -207,6 +225,29 @@ class SoundScene {
       });
       return button;
     });
+
+    const export_button = document.getElementById("export");
+    if (export_button instanceof HTMLButtonElement) {
+      export_button.disabled = false;
```

Comment:
Really this should updated every time a score is selected, not at the scene start.